addons/event/models/event_event.py

```diff
@@ -88,6 +90,16 @@ class EventEvent(models.Model):
     _inherit = ['mail.thread', 'mail.activity.mixin']
     _order = 'date_begin'
 
+    @api.model
+    def default_get(self, default_fields):
+        result = super().default_get(default_fields)
+        if not result.get('date_begin'):
+            now = fields.Datetime.now()
+            result['date_begin'] = now + (now.min - now) % timedelta(minutes=30)
```

Comment:
A bit strange, not sure how it's working, did you find this somewhere else?

Would be nice to write a small test.

Reply:
I think I prefer that one, along with a small python comment that explains we are rounding to nearest half hour.

Also, would you mind renaming "default_fields" to "fields_list" in the method arguments?
It seems like the original models.py definition is `def default_get(self, fields_list):`.

After that, we should be good to merge :rocket: 

Cheers!